tests/test_tokenization_mistral_common.py

```diff
@@ -0,0 +1,607 @@
+import tempfile
+import unittest
+
+from mistral_common.exceptions import InvalidMessageStructureException, TokenizerException
+
+from transformers.models.auto.tokenization_auto import AutoTokenizer
+from transformers.tokenization_mistral_common import MistralCommonTokenizer
+
+
+class TestMistralCommonTokenizer(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.tekken_tokenizer: MistralCommonTokenizer = AutoTokenizer.from_pretrained(
+            "mistralai/Mistral-Small-3.1-24B-Instruct-2503", tokenizer_type="mistral-common"
+        )
+        cls.spm_tokenizer: MistralCommonTokenizer = AutoTokenizer.from_pretrained(
+            "mistralai/Mistral-Small-Instruct-2409", tokenizer_type="mistral-common"
+        )
```

Comment:
I think testing vs the actual mistral-common api might be better 😉

Reply:
You're right ! I pushed changes doing so